build_msvc/bitcoin_config.h

```diff
@@ -137,18 +137,6 @@
    don't. */
 #define HAVE_DECL_STRNLEN 1
 
-/* Define to 1 if you have the declaration of `__builtin_clz', and to 0 if you
-   don't. */
-//#define HAVE_DECL___BUILTIN_CLZ 1
```

Comment:
These were 1; should they be replaced with the corresponding true define? I'm a bit surprised MSVC has the exact same builtins though.

Reply:
They were 1, but commented out. I think they were only in here because `build_msvc/bitcoin_config.h` began as a copy-paste job from `src/config/bitcoin-config.h`. MSVC doesn't have any of these builtins.

Reply:
That makes more sense, thanks.